schema/mysql-migrations/upgrade_188.sql

```diff
@@ -14,4 +21,5 @@ ADD CONSTRAINT director_generated_config_activity
 
 INSERT INTO director_schema_migration
   (schema_version, migration_time)
-  VALUES (188, NOW());
+  VALUES (188, NOW())
+  ON DUPLICATE KEY UPDATE schema_version = 188, migration_time = NOW();
```

Comment:
Is that really necessary? For users without this issue, the version has already been updated and the unique index applied. Those who manually removed duplicates also have the version updated and index applied. However, for users awaiting a fix, the upgrade script exits before updating the version.

Reply:
Makes sense, I will remove this.